web/template/your_legal_rights_and_responsibilities_general.gohtml

```diff
@@ -7,7 +7,7 @@
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-xl">{{ tr .App "yourLegalRightsAndResponsibilities" }}</h1>
 
-      {{ trHtml .App "yourLegalRightsAndResponsibilitiesContent" }}
+      {{ trHtml .App "yourLegalRightsAndResponsibilitiesContentPfa" }}
```

Comment:
I've left this as the existing PFA wording for now - will raise with content that there's a version viewable when the user is logged out so no way of telling the LPA type.

Reply:
I hate this page existing, really should be something we tackle later when we do the "help & support" bit

Reply:
I flagged in the ticket that it may be worth dropping the link from the footer so we'll see what people think.